cmd/webhook_remove_test.go

```diff
@@ -0,0 +1,423 @@
+			}
+			if gotDryRun != tt.wantDryRun {
+				t.Errorf("removeWebhookFlagCheck() gotDryRun = %v, want %v", gotDryRun, tt.wantDryRun)
+			}
+		})
+	}
+}
+
+func Test_removeWebhookCommand(t *testing.T) {
+	originalConfig := config
+
+	httpmock.Activate()
+
+	defer func() {
+		httpmock.DeactivateAndReset()
+
+		config = originalConfig
+	}()
+
+	config.Org = MockRepoName
+
+	type args struct {
+		cmd  *cobra.Command
+		repo *repository
+	}
+
+	cmdInvalidFlags := &cobra.Command{Use: "webhook-remove"}
+
+	cmdDryRunOnFlags := &cobra.Command{Use: "webhook-remove"}
+	cmdDryRunOnFlags.Flags().BoolP("dry-run", "d", true, "dry run flag")
+	cmdDryRunOnFlags.Flags().StringP("host", "", "https://some-external-webhook.com", "host flag")
+	cmdDryRunOnFlags.Flags().StringP("repos", "", "filepath", "repos flag")
+
+	cmdDryRunOffFlags := &cobra.Command{Use: "webhook-remove"}
+	cmdDryRunOffFlags.Flags().BoolP("dry-run", "d", false, "dry run flag")
+	cmdDryRunOffFlags.Flags().StringP("host", "", "https://some-external-webhook.com", "host flag")
+	cmdDryRunOffFlags.Flags().StringP("repos", "", "filepath", "repos flag")
+
+	tests := []struct {
+		name         string
+		args         args
+		mockHTTPFunc func()
+		wantErr      bool
+	}{
+		{
+			name: "removeWebhookCommand flag check failure",
+			args: args{
+				cmd: cmdInvalidFlags,
+			},
+			mockHTTPFunc: func() {},
+			wantErr:      true,
+		},
+		{
+			name: "removeWebhookCommand repo read failure",
+			args: args{
+				cmd: cmdDryRunOnFlags,
+				repo: &repository{
+					reader: &mockRepositoryReader{
+						readFail: true,
+					},
+				},
+			},
+			mockHTTPFunc: func() {},
+			wantErr:      true,
+		},
+		{
+			name: "removeWebhookCommand dry run success",
+			args: args{
+				cmd: cmdDryRunOnFlags,
+				repo: &repository{
+					reader: &mockRepositoryReader{
+						returnValue: []string{
+							"some-repo-name",
+						},
+					},
+				},
+			},
+			mockHTTPFunc: func() {},
+			wantErr:      false,
+		},
+		{
+			name: "removeWebhookCommand remove webhook error",
+			args: args{
+				cmd: cmdDryRunOffFlags,
+				repo: &repository{
+					reader: &mockRepositoryReader{
+						returnValue: []string{
+							"some-repo",
+						},
+					},
+				},
+			},
+			mockHTTPFunc: func() {
+				mockHTTPResponder(
+					"GET",
+					"/repos/some-org/some-repo/hooks",
+					"testdata/mockGetWebhooksResponse.json",
+					200,
+				)
+				mockHTTPResponder(
+					"DELETE",
+					"/repos/some-org/some-repo/hooks/123",
+					"testdata/mockDeleteWebhookResponse404.json",
+					404,
+				)
+			},
+			wantErr: true,
+		},
+		{
+			name: "removeWebhookCommand success with empty repo",
+			args: args{
+				cmd: cmdDryRunOffFlags,
+				repo: &repository{
+					reader: &mockRepositoryReader{
+						returnValue: []string{},
+					},
+				},
+			},
+			mockHTTPFunc: func() {},
+			wantErr:      false,
+		},
+		{
+			name: "removeWebhookCommand success with remove",
```

Comment:
What does this test name meen?
can i be more descriptive ie. "removeWebhookCommand returns Sucess on given repo?